NebulaModel/Utils/TimeUtils.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace NebulaModel.Utils
+{
+    public class TimeUtils
+    {
+        static readonly DateTime UNIX_EPOCH = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+        public static long CurrentUnixTimestampMilliseconds()
```

Comment:
`DateTimeOffset.Now.ToUnixTimeSeconds()` now exists :)

Bit of an easier option than doing all this.

Reply:
Doesn't exist since we are in 3.5 :(